### ISSUE TITLE
Make Akilang self-hosting

### ISSUE THREAD

**Issue by syegulalp** [syegulalp/Akilang#1]
If you're going to dream, dream big.

**Comment by Aareon** [comment]
By this I am assuming you will also implement an LL IR compiler in Aki as well?

**Comment by syegulalp** [comment]
That's a longer shot than I originally imagined! My not-so-impossible idea is to take the existing llvmlite binaries, which I depend on anyway, and have the Aki-implemented compiler use that for codegen -- essentially, replace the Python code with Aki code as completely as possible.

**Comment by Aareon** [comment]
I would suggest studying the LL IR that is currently being emitted and do your best to write an emitter that emits the same, or better, IR in Aki.

**Comment by syegulalp** [comment]
Yes, it would basically be the `llvmlite` library by way of Aki.
